src/js/main.js

```diff
@@ -212,7 +217,46 @@ Site.Map = {
 
     _this.panning = true;
 
+    _this.startPanEvent();
+
     window.requestAnimationFrame(_this.pan.bind(_this));
+
+  },
+
+  startPanEvent: function() {
+    var _this =  this;
+
+    var startPanEvent = new CustomEvent('startpan', {
```

Comment:
doesnt actually work in IE (https://caniuse.com/#search=CustomEvent) but I guess we don't really care